apis/r/tests/testthat/test-write-soma-resume.R

```diff
@@ -29,6 +29,8 @@ test_that("Factory re-creation", {
     expect_error(fxn(uri))
     obj$close()
   }
+
+  gc()
```

Comment:
FWIW there was never conclusive proof these help.  There is IIRC a decade+ old post of mine arguing to chain several calls to `gc()` ... but at the end of the day the R memory management layer is (as for any other managed languages) a black box.  So :man_shrugging: -- this may not do harm but it most likely won't help much.